src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs

```diff
@@ -16,6 +15,29 @@ internal class QuickInfoDisplayDeferredContent : IDeferredQuickInfoContent
         public IDeferredQuickInfoContent ExceptionText { get; }
         public IDeferredQuickInfoContent WarningGlyph { get; }
 
+        // DO NOT REMOVE: compat for Typescript
+        public QuickInfoDisplayDeferredContent(
```

Comment:
Isn't this constructor and the one below essentially the same parameter wise?

Reply:
There's one less parameter. :-)